src/views/Config/Config.js

```diff
@@ -0,0 +1,180 @@
+import React, {Component} from 'react'
+import {card as cardStyle, hr as hrStyle} from './utils/styles';
+
+import UserCard from './userCard'
+import SocialButton from "./SocialButton";
+
+class Config extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            logged: false,
+            user: {},
+            currentProvider: ''
+        }
+        this.nodes = {};
+
+        this.onLoginSuccess = this.onLoginSuccess.bind(this);
+        this.onLoginFailure = this.onLoginFailure.bind(this);
+        this.onLogoutSuccess = this.onLogoutSuccess.bind(this);
+        this.onLogoutFailure = this.onLogoutFailure.bind(this);
+        this.logout = this.logout.bind(this)
+    }
+
+    setNodeRef(provider, node) {
+        if (node) {
+            this.nodes[provider] = node
```

Comment:
What is this doing?

Reply:
keeps the reference to Providers object in our case we have 3